pkg/gather/clusterconfig/infrastructures.go

```diff
@@ -44,7 +44,6 @@ func gatherClusterInfrastructure(ctx context.Context, configClient configv1clien
 }
 
 func anonymizeInfrastructure(config *configv1.Infrastructure) *configv1.Infrastructure {
-	config.Status.EtcdDiscoveryDomain = anonymize.AnonymizeURL(config.Status.EtcdDiscoveryDomain)
```

Comment:
The `config.Status.EtcdDiscoveryDomain` is deprecated since 4.7. See the source https://github.com/openshift/api/blob/38058be7d6ef61b980f52f3633c41bbf8e20a0ea/config/v1/types_infrastructure.go#L68